### ISSUE TITLE
SelectBox: Add describedby support for details in Voiceover

### ISSUE THREAD

**Issue by cleidigh** [Microsoft/vscode#60429]
Voiceover does not read new option details because of describedby issue.

May require the new ARIA combo box model.

**Comment by cleidigh** [comment]
@bpasero 
Any problem was me creating new tags for NVDA and voiceover like there is already a tag for jaws?

Accessibility-NVDA
Accessibility-Voiceover

I will also volunteer to do triage on accessibility issues if I can identify them.
